bip78/src/receiver/mod.rs

```diff
@@ -16,7 +14,7 @@ pub struct UncheckedProposal {
 }
 
 impl UncheckedProposal {
-    pub fn from_request(body: impl std::io::Read, query: &str, headers: impl Headers) -> Result<Self, RequestError> {
+    pub fn from_request(body: impl std::io::Read, _query: &str, headers: impl Headers) -> Result<Self, RequestError> {
```

Comment:
`query` MUST be processed - there is important information such as `disableoutputsubstitution`.

Reply:
I would have removed the parameter if thinking it doesn't have to be processed, but yeah maybe I should have added a TODO as a reminder